var/spack/repos/builtin/packages/hdf5-vol-log/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hdf5VolLog(AutotoolsPackage):
+    """Log-based VOL - an HDF5 VOL Plugin that stores HDF5 datasets in a log-based
+    storage layout."""
+
+    homepage = 'https://github.com/DataLib-ECP/vol-log-based'
+    url      = 'https://github.com/DataLib-ECP/vol-log-based'
+    git = 'https://github.com/DataLib-ECP/vol-log-based.git'
+    maintainers = ['hyoklee']
+
+    version('master', branch='master', git=git,
+            commit='b13778efd9e0c79135a9d7352104985408078d45')
+
+    depends_on('hdf5@1.12.1')
```

Comment:
We don't usually pin a specific version but give at least a range of valid versions, for instance:
```suggestion
    depends_on('hdf5@1.12.1:')
```